go/vt/vtexplain/testdata/twopc-output/unsharded-output.txt

```diff
@@ -12,7 +12,7 @@ select * from t1
 insert into t1 (id,intval,floatval) values (1,2,3.14)
 
 1 ks_unsharded/-: begin
-1 ks_unsharded/-: insert into t1(id, intval, floatval) values (1, 2, 3.14)
+1 ks_unsharded/-: insert into t1(id, intval, floatval) values (1, 2, cast(3.14 as DECIMAL(3, 2)))
```

Comment:
@systay Is this one correct now that we only emit the cast if the scale is 0?

Reply:
hmm... weird. these tests were not running, so our changes for the format were not picked up